custom_conf.py

```diff
@@ -84,7 +84,8 @@
     'mattermost': 'https://chat.canonical.com/canonical/channels/documentation',
 
     # Change to the GitHub URL for your project
-    'github_url': 'https://github.com/canonical/starter-pack',
+    # 'github_url': 'https://github.com/canonical/starter-pack',
+    'github_url': 'https://github.com/canonical/sphinx-docs-starter-pack',
```

Comment:
```suggestion
    'github_url': 'https://github.com/canonical/sphinx-docs-starter-pack',
```

Good catch!
No need to keep the old URL though, you can just delete it. :)